benchmark/Makefile.am

```diff
@@ -35,9 +39,10 @@ $(LOCAL_DIR)/config.h: $(LOCAL_DIR)
 
 $(BENCHOUT): $(LOCAL_DIR)/config.h
 	cd $(LOCAL_DIR) && make clean
-	cd $(LOCAL_DIR) && $(TIMER) make
+	cd $(LOCAL_DIR) && $(TIMER) -o run make
```

Comment:
Name it something like `baseline.times` , the other one correspondingly and remember to delete them on `clobber`

Reply:
The corresponding baseline.times and build-recorder.times are generated inside $(LOCAL_DIR), so since clobber is removing it entirely, they are as well.